tests/csdmpy_test.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pytest
+
+import csdmpy as cp
+
+
+def test_append():
```

Comment:
may be better to call this function `test_join` to better mirror the function name